app/backend/src/couchers/servicers/donations.py

```diff
@@ -112,6 +112,10 @@ def Webhook(self, request, context):
                 # amount comes in cents
                 amount = int(data_object["amount"]) // 100
                 receipt_url = data_object["receipt_url"]
+
+                # may be check for amount to enable phone verify
+                session.add(User(has_donated=True))
```

Comment:
This syntax creates a new user with just this flag set. The SQLalchemy ORM loads the user and will automatically update it so you just need to do `user.has_donated = True`